Multisig/Assets.xcassets/ico-confirm-thin.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
```

Comment:
This one is exactly same as checkmark.circle right?

Reply:
not exactly. It is thinner and has small margins 🤷‍♂️

Reply:
Ah I see, If you want let's make sure if we can use the same icon